app/Http/Controllers/Tabbar/TabbarController.php

```diff
@@ -3,39 +3,34 @@
 namespace App\Http\Controllers\Tabbar;
 
 use App\Http\Controllers\Controller;
-use App\Settings\PathInfo;
+use Illuminate\Http\Request;
+use App\Http\Services\Tabbar\TabbarService;
 
-class TabbarController extends Controller
-{
-    /*
-    |--------------------------------------------------------------------------
-    | TabbarController
-    |--------------------------------------------------------------------------
-    |
-    | Tabbarコントローラ
-    |
-    */
+/**
+ * TabbarController
+ * 
+ * ページの下に表示しているタブバーを扱う
+ */
+class TabbarController extends Controller {
```

Comment:
__[code_sniffer]__
* Opening brace of a class must be on the line after the definition